presto-vertica/src/test/java/io/prestosql/plugin/vertica/VerticaQueryRunner.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.vertica;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logger;
+import io.airlift.log.Logging;
+import io.prestosql.Session;
+import io.prestosql.plugin.tpch.TpchPlugin;
+import io.prestosql.testing.DistributedQueryRunner;
+import io.prestosql.testing.QueryRunner;
+import io.prestosql.tpch.TpchTable;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.airlift.testing.Closeables.closeAllSuppress;
+import static io.prestosql.plugin.tpch.TpchMetadata.TINY_SCHEMA_NAME;
+import static io.prestosql.testing.QueryAssertions.copyTpchTables;
+import static io.prestosql.testing.TestingSession.testSessionBuilder;
+
+public final class VerticaQueryRunner
+{
+    private VerticaQueryRunner() {}
+
+    private static final String TPCH_SCHEMA = "tpch";
+
+    public static QueryRunner createVerticaQueryRunner(TestingVerticaServer server, TpchTable<?>... tables)
```

Comment:
I think  it's ok to inline.
The added maintanance overheader of uipdating an overload is not worth it, when you have just a couple of call places.